gcp/appengine/frontend_handlers.py

```diff
@@ -40,7 +43,8 @@
 _PAGE_SIZE = 16
 _PAGE_LOOKAHEAD = 4
 _REQUESTS_PER_MIN = 30
-_WORD_CHARACTERS_OR_DASH = re.compile(r'^[\w-]+$')
+_WORD_CHARACTERS_OR_DASH = re.compile(r'^[\w\-]+$')
```

Comment:
Based on the explanation regex101.com gives me, I'm not seeing the need for escaping the hyphen, are you doing this for a particular reason? Same question for the next line.

Reply:
+1 to @oliverchang's comment about comments :-)

Reply:
You're right it's not needed to be escaped, but if you want to add one more character it becomes a range (e.g. `[A-Z]`), so I just added `\` to be consistent with the other regex. The other one's removed now so I reverted this.